cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/TimeSeriesProfileParserColumnar.java

```diff
@@ -21,6 +21,8 @@
 @JsonDeserialize(builder = TimeSeriesProfileParserColumnar.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+// Uses custom serializer to handle polymorphic serialization
+// See issue: https://github.com/FasterXML/jackson-databind/issues/2185
```

Comment:
use javadocs as they can `@link` to other classes

Reply:
Moved comments into javadocs